pathd/path_pcep_controller.c

```diff
@@ -710,6 +713,7 @@ int pcep_thread_event_pathd_event(struct ctrl_state *ctrl_state,
 
 	for (i = 0; i < ctrl_state->pcc_count; i++) {
 		struct pcc_state *pcc_state = ctrl_state->pcc[i];
+		path->is_delegated = is_best_pce(ctrl_state, i);
```

Comment:
We want the pcc to do this, the controller shouldn't be responsible for that. The controller may elect a pcc and call some function like `pcep_pcc_set_primary(bool is_primary)` and then the pcc do the path customization when required.

Reply:
will change the handle of which is delegated from controller to pcc